spec/support/shared_examples/proper_pagination_params.rb

```diff
@@ -26,4 +26,20 @@
   it 'responds with next_page' do
     expect(response_body(response)['next_page']).to be pagination_params[:next_page]
   end
+
+  get_page_from_uri = lambda { |uri|
+    uri ? Rack::Utils.parse_query(URI.parse(uri).query)['page'].to_i : nil
+  }
```

Comment:
Can we move this method to a helper? In specs we shouldn't have methods :disappointed: 
```rb
# spec/support/helpers/uri_parser.rb
class UriParser
  def get_page_from(uri)
    return nil if uri.nil?
    
    Rack::Utils.parse_query(URI.parse(uri).query)['page'].to_i
  end
end
```